meta/classes/nfsroot.bbclass

```diff
@@ -4,7 +4,7 @@ python nfsroot_eventhandler() {
         return
 
     # Set the path for the Unix socket
-    socket_path = '/tmp/nfsup.sock'
+    socket_path = f"/tmp/nfsup-{os.environ['USER']}.sock"
```

Comment:
`os` needs to be imported before it's used.

Reply:
Indeed. It's there but a line too late. Fixed